packages/nimble-components/src/theme-provider/design-token-comments.ts

```diff
@@ -70,6 +70,7 @@ export const comments: { readonly [key in TokenName]: string | null } = {
     dialogLargeWidth: 'Standard width for large dialogs.',
     dialogLargeHeight: 'Standard height for large dialogs.',
     dialogLargeMaxHeight: 'Standard maximum height for large dialogs.',
+    menuMinWidth: 'Standard menu min width for menu popup.',
```

Comment:
minWidth needs to be added to the suffix list, can you do it similar to maxHeight as discussed here: https://github.com/ni/nimble/pull/1553#discussion_r1374822884

Reply:
Added as mentioned in the linked comment